examples/pi.cxx

```diff
@@ -9,11 +9,12 @@ int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
   typedef RAJA::seq_reduce reduce_policy;
```

Comment:
What do folks think about switching over to the C++11 'using reduce_policy = RAJA::seq_reduce' (for example) for of this sort of thing?  I've taken a liking to it....